packages/telemetry/src/slog-to-otel.js

```diff
@@ -552,15 +561,31 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
         break;
       }
       case 'cosmic-swingset-begin-block': {
+        if (spans.topName() === `intra-block`) {
+          spans.pop(`intra-block`);
+        }
```

Comment:
This introduces a new span to capture the duration between "commit-block" and the next "begin-block", and layers that as a child of the "previous" block, which causes the block's duration to now measure the block to block time.

Before we were not capturing this gap anywhere, causing blocks to appear shorter than they really were.

cc @mfig @warner @arirubinstein